Philips.CodeAnalysis.MaintainabilityAnalyzers/Maintainability/LogExceptionAnalyzer.cs

```diff
@@ -90,7 +90,7 @@ public void AnalyzeCatchException(SyntaxNodeAnalysisContext context)
 				var catchNode = (CatchClauseSyntax)context.Node;
 				// Look for logging method calls underneath this node.
 				var hasCallingLogNodes = catchNode.DescendantNodes()
-					.OfType<InvocationExpressionSyntax>().Any(x => IsCallingLogMethod(context, x));
+					.OfType<InvocationExpressionSyntax>().Any(x => IsCallingLogMethod(x));

```

Comment:
Consider to have every Linq statement on a separate line for readability

Reply:
Fixed, and filed Issue to explore automation